### ISSUE TITLE
test: flakiness TestFullReconstructFromBridge

### ISSUE THREAD

**Issue by vgonkivs** [celestiaorg/celestia-node#943]
```
--- PASS: TestFullReconstructFromBridge (1.26s)
panic: Fail in goroutine after TestFullReconstructFromBridge has completed
goroutine 14064 [running]:
testing.(*common).Fail(0xc0020f76c0)
          /opt/hostedtoolcache/go/1.18.4/x64/src/testing/testing.go:817 +0xe5
testing.(*common).Errorf(0xc0020f76c0, {0x213d5c8?, 0xc0020f76c0?}, {0xc00452e6b0?, 0x47822f?, 0xc0009fcb90?})
          /opt/hostedtoolcache/go/1.18.4/x64/src/testing/testing.go:934 +0x65
github.com/stretchr/testify/assert.Fail({0x2db0e20, 0xc0020f76c0}, {0xc003ffe240, 0x84}, {0x0, 0x0, 0x0})
          /home/runner/go/pkg/mod/github.com/stretchr/testify@v1.8.0/assert/assertions.go:264 +0x384
github.com/stretchr/testify/assert.NoError({0x2db0e20, 0xc0020f76c0}, {0x2db0b40, 0xc004e81200}, {0x0, 0x0, 0x0})
          /home/runner/go/pkg/mod/github.com/stretchr/testify@v1.8.0/assert/assertions.go:1392 +0x105
github.com/stretchr/testify/require.NoError({0x2db8948, 0xc0020f76c0}, {0x2db0b40, 0xc004e81200}, {0x0, 0x0, 0x0})
          /home/runner/go/pkg/mod/github.com/stretchr/testify@v1.8.0/require/require.go:1261 +0x96
github.com/celestiaorg/celestia-node/node/tests/swamp.(*Swamp).SubmitData(0xc0015c5440, {0x2dcb940, 0xc002a94c60}, 0x0?, {0xc002852000, 0x1000, 0x0?})
         /home/runner/work/celestia-node/celestia-node/node/tests/swamp/swamp_tx.go:19 +0x105
github.com/celestiaorg/celestia-node/node/tests/swamp.(*Swamp).FillBlocks(0xc0015c5440, {0x2dcb940, 0xc002a94c60}, 0xc0029f67d0?, 0x10, 0x14)
	/home/runner/work/celestia-node/celestia-node/node/tests/swamp/swamp_tx.go:31 +0x177
created by github.com/celestiaorg/celestia-node/node/tests.TestFullReconstructFromBridge
	/home/runner/work/celestia-node/celestia-node/node/tests/reconstruct_test.go:48 +0x14b
FAIL	github.com/celestiaorg/celestia-node/node/tests	13.089s
```

**Comment by vgonkivs** [comment]
This test should be re-designed in order to avoid flakiness.

**Comment by Bidon15** [comment]
Can be the same root cause 
Ref: https://github.com/celestiaorg/celestia-node/issues/985

**Comment by vgonkivs** [comment]
fixed within #1000
